### ISSUE TITLE
CW-Toots display images

### ISSUE THREAD

**Issue by truhe2342** [tootsuite/mastodon#8260]
Currently, when creating a toot with a cw images are automatically (forced) to being sensitive. This results in toots consisting of:

* The CW headline
* The expand button
* The long text (which becomes visible when clicking the expand button)
* The sensitive image (which is always visible)

**The sensitive image is always visible. It is either hidden beneath the sensitive warning _or automatically shown_, when the user has activated „Always show media marked as sensitive“ in the profile.**

CW and the sensitive mark are two different things.

1. When activating the CW the image should also be hidden and only displayed after clicking the expand button.
2. When creating a toot with a CW the image should not automatically be forced to be sensitive.

**Comment by truhe2342** [comment]
Thanks. I did search before posting my issue, but obviously dug not deep enough ;).

**Comment by Cassolotl** [comment]
Oh and also *CWs should not force a “NSFW” state #4689*, which is more what you're asking for!

**Comment by ThibG** [comment]
Yeah, I would also like if attachments were put below the CW, like in glitch-soc.
It would also help with the awkward thing going on with CWs becoming text if the toot has no text, thus breaking copying CW on reply.

**Comment by tastytea** [comment]
Users of other Fediverse software, such as Misskey and Pleroma, don't expect that media is shown when a CW is used, because they are hidden on these platforms. That's why they don't set the media to sensitive when they use a CW. Even if they know that Mastodon displays them, it is hard to remember it every time.

Misskey has worked around that and sets the sensitive flag when federating but Pleroma does not. I'm hearing complaints from Mastodon users now and then that they see sensitive media on CW-ed and collapsed toots in their timelines.

I believe there is no downside to hiding the media along with the text.
